ljepotaservis/ljepotaservis.Web/ClientApp/src/components/authentication/Login.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import { LinkContainer } from "react-router-bootstrap";
+import { FormGroup, FormControl, ControlLabel, Button } from "react-bootstrap";
+
+export default class Login extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      email: "",
+      password: "",
+      loginError: ""
+    };
+  }
+
+  handleChange = e => {
```

Comment:
I think this argument should be named event

Reply:
Okay, will change this